server/inngest/eventHandlers/sendUpcomingNotifications.ts

```
@@ -0,0 +1,102 @@
+          scheduledEndAt: sub.schedule.endAt,
+          scheduledDate: new Date(sub.schedule.startAt).toLocaleDateString('pt-BR'),
+          scheduledTime: new Date(sub.schedule.startAt).toLocaleTimeString('pt-BR', { hour: '2-digit', minute: '2-digit' })
+        }))
+      };
+
+      await $fetch(`${hemocioneIdBaseUrl}/notifications/upcoming-event`, {
+        method: "POST",
+        body: {
+          targets: { userIds },
+          channels: {
+            push: { 
```

ESLint

[error] Delete `·`

(prettier/prettier)